src/wirecloud/commons/static/js/StyledElements/Expander.js

```diff
@@ -79,9 +79,9 @@
         if (this.toggleButton) {
             this.toggleButton.addEventListener('click', callback);
         }
-        if (options.listenOnTitle) {
-            this.titleContainer.wrapperElement.style.cursor = "pointer";
-            this.titleContainer.wrapperElement.addEventListener('click', callback, false);
+        if (options.listenOnHeader) {
+            header.style.cursor = "pointer";
+            header.addEventListener('click', callback, false);
```

Comment:
My suggestion is to allow clicking on the full header/row instead of only the title. That way users have a bigger area to click.
But I can easily revert this if it is not desired.

Reply:
I think your approach is much better than the current one but I think we must maintain the `listenOnHeader` option name.

Reply:
Okay, so I keep the option name but still make the full header clickable, right?

Reply:
Yeah, the idea is to maintain the option name, but changing the behaviour to the one provided in your PR.